tools/gfx/vulkan/render-vk.cpp

```diff
@@ -826,7 +829,16 @@ class VKDevice : public RendererBase
                         }
                         descriptorSetInfo.vkBindings.add(vkBindingRangeDesc);
                     }
+                    VkDescriptorSetLayoutCreateInfo createInfo = {};
+                    createInfo.sType = VK_STRUCTURE_TYPE_DESCRIPTOR_SET_LAYOUT_CREATE_INFO;
+                    createInfo.pBindings = descriptorSetInfo.vkBindings.getBuffer();
+                    createInfo.bindingCount = (uint32_t)descriptorSetInfo.vkBindings.getCount();
+                    VkDescriptorSetLayout vkDescSetLayout;
+                    SLANG_RETURN_ON_FAIL(m_renderer->m_api.vkCreateDescriptorSetLayout(
```

Comment:
I had to go and convince myself that applying `SLANG_RETURN_ON_FAIL` to a Vulkan API call actually works, but the documentation for `VkResult` is consistent with `HRESULT` and our Slang/`gfx` `Result` in that non-negative values represent success. That's kind of convenient/cool.